Automattic/wp-calypso
# move discover nav higher in componentry

Related to #

## Proposed Changes

* Refactors the discover nav to be higher than the stream component it controls in the heirarchy. This resolves issues with parent re-renders effecting scroll position and other issues while loading and makes more sense generally.

## Testing Instructions



* Run this calypso branch.
* Test the discover navigation and verify it works as expected.
* Reload the /discover page, start clicking on the scroll arrows, and verify there is no longer any odd clunky behavior of the navigation while items are still loading.

## Pre-merge Checklist
